### ISSUE TITLE
inheritance graphs: problem when a field is inherited in two other fiels

### ISSUE THREAD

**Issue by maieul** [plk/biber#117]
Considering this MWE

`````
\documentclass{article}
\usepackage[style=verbose]{biblatex}
\usepackage{filecontents}
\begin{filecontents}{test.bib}
@book{Main,
  Author = {Someone},
  Title={Some title}
}

@bookinbook{Second,
  Title={Some title},
  Crossref={Main},
}
\end{filecontents}
\bibliography{test.bib}
\begin{document}
\nocite{*}
\printbibliography
\end{document}
`````

The  `author` of the `second` entry is inherited from the `author` of `bookauthor`. 

However, the .dot files produced by `-output-format=dot --dot-include=crossref,field` does not mention it.

**Comment by plk** [comment]
Well spotted, should be fixed now in 2.5 dev.

**Comment by maieul** [comment]
it works. Thanks.
